dtests/src/test/java/io/snappydata/hydra/cdcConnector/SnappyCDCTest.java

```diff
@@ -1,41 +1,359 @@
 package io.snappydata.hydra.cdcConnector;
 
-import hydra.Log;
-
-import io.snappydata.hydra.cluster.SnappyTest;
-
 import java.io.*;
-
 import java.net.InetAddress;
+import java.rmi.RemoteException;
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.ResultSet;
-import java.util.List;
+import java.sql.SQLException;
+import java.util.*;
+
+import com.gemstone.gemfire.GemFireConfigException;
+import hydra.HydraRuntimeException;
+import hydra.Log;
+import hydra.RemoteTestModule;
+import io.snappydata.hydra.cluster.SnappyBB;
+import io.snappydata.hydra.cluster.SnappyStartUpTest;
+import io.snappydata.hydra.cluster.SnappyTest;
+import org.apache.commons.io.FileUtils;
 
 public class SnappyCDCTest extends SnappyTest {
-  protected static SnappyCDCTest snappyCDCTest;
+  public static SnappyCDCTest snappyCDCTest;
 
   public SnappyCDCTest() {
   }
 
-
-  public static Connection getConnections() {
+  public Connection getSnappyConnection() {
     Connection conn = null;
     List<String> endpoints = validateLocatorEndpointData();
-    String hostPort = endpoints.get(0);
-    String url = "jdbc:snappydata://" + hostPort;
-    Log.getLogWriter().info("url is " + url);
+    String url = "jdbc:snappydata://" + endpoints.get(0);
+    Log.getLogWriter().info("The URL is " + url);
     String driver = "io.snappydata.jdbc.ClientDriver";
     try {
       Class.forName(driver);
       conn = DriverManager.getConnection(url);
     } catch (Exception ex) {
-      System.out.println("Caught exception in getConnection() method" + ex.getMessage());
+      System.out.println("Caught exception in getSnappyConnection() method" + ex.getMessage());
     }
-
     return conn;
   }
 
+  public static List getHostNames() {
+    Set<String> pidList;
+    pidList = SnappyStartUpTest.getServerPidList();
+    List asList = new ArrayList(pidList);
+    return asList;
+  }
+
+  public void meanKillProcesses() {
+    Process pr = null;
+    ProcessBuilder pb;
+    File logFile, log = null, serverKillOutput;
+    Set<String> pidList;
+    Boolean isStopStart = SnappyCDCPrms.getIsStopStartCluster();
+    String snappyPath = SnappyCDCPrms.getSnappyFileLoc();
+    String nodeType = SnappyCDCPrms.getNodeType();
+    int numNodes = SnappyCDCPrms.getNumNodes();
+    String pidString;
+    try {
+      pidList = SnappyStartUpTest.getServerPidList();
```

Comment:
Create a separate method for the common code to be used for meankill and use the same.